Source/CarthageKit/Resolver.swift

```diff
@@ -163,9 +164,10 @@ public struct Resolver {
 	/// the specified node (or as a root otherwise).
 	///
 	/// This is a helper method, and not meant to be called from outside.
-	private func graphs(for nodes: [DependencyNode], dependencyOf: DependencyNode?, basedOnGraph inputGraph: DependencyGraph) -> SignalProducer<DependencyGraph, CarthageError> {
+	private func graphs(for nodes: [DependencyNode], dependencyOf: DependencyNode?,
+	                    basedOnGraph inputGraph: DependencyGraph) -> SignalProducer<DependencyGraph, CarthageError> {
```

Comment:
That is the Swift Standard Library style. We use it in ReactiveSwift and ReactiveCocoa too,